src/utils/logger.js

```diff
@@ -1,6 +1,20 @@
 const dotenv = require('dotenv')
 dotenv.config()
-const logger = require('pino')({
+const pino = require('pino')
+const hooks = {
+  logMethod(args, method) {
+    if (args.length > 1) {
+      for (let i = 1; i < args.length; i++) {
+        if (typeof args[i] === 'object')
+          args[i] = JSON.stringify(args[i], null, 2)
```

Comment:
Should we add a try catch block here in case the object cannot be stringified?

Reply:
Nice catch! This will only happen if the object has a circular structure.
Example:
```js
let a = {b:1,c:1};
a.a = a; // references itself
JSON.stringify(a) // this will throw an error because the object has a circular structure
```
Or an array containing itself:
```js
var arr = [];
arr[0] = arr;
JSON.stringify(a) // this will throw an error because the object has a circular structure
```
I'll add a try-catch block.